actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/datasecurity/impl/DefaultDecryptionServiceImpl.java

```diff
@@ -105,6 +105,8 @@ public static String decrypt(String value, boolean throwExceptionOnFailure) {
       }
       return dValue;
     } catch (Exception ex) {
+      ProjectLogger.log(
+          "DefaultDecryptionServiceImpl:decrypt:value " + value, LoggerEnum.INFO.name());
```

Comment:
Should we print throwExceptionOnFailure also in the logs like below:
"DefaultDecryptionServiceImpl:decrypt: "+value+" - "+throwExceptionOnFailure+" - Exception occurred with error message = "

Reply:
As I indicated in this PR description, will like to remove this line or retain - based on results. Therefore it is 2 lines. Pls consider this as an 'investigative' PR in environment.

Reply:
ok